content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -55,7 +48,13 @@ <h3>Summary:</h3>
         <div class="row">
           <div class="col-md-12">
             {{< note >}}
-            <p>If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section</a>, you may have deleted the Service exposing the Deployment. In that case, please expose the Deployment again using the following command:</p><p><code><b>kubectl expose deployment/kubernetes-bootcamp --type="NodePort" --port 8080</b></code></p>
+              <p><b>NOTE</b> If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section </a>, then you
```

Comment:
The "NOTE" text can be removed from here since the shortcode automatically includes it. This will prevent it from being duplicated.

```suggestion
              <p>If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section </a>, then you
```